seed_labeling.ipynb

```diff
@@ -1,732 +1,1020 @@
 {
```

Comment:
<p>rerun so this doedsn't appear</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>